apps/code-examples/src/app/code-examples/pages/page/home-page-blocks-layout-demo/demo.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { SkyIconModule } from '@skyux/icon';
+import { SkyPageModule } from '@skyux/pages';
+
+import { HomePageContentComponent } from './home-page-content.component';
+
+@Component({
+  standalone: true,
+  selector: 'app-demo',
+  templateUrl: './demo.component.html',
+  imports: [HomePageContentComponent, SkyIconModule, SkyPageModule],
+})
+export class DemoComponent {
+  protected readonly recentLinks = [
```

Comment:
Is there a type we can import and assign to this property to better assist with consumers interacting with the code examples?

Reply:
Good call. Added the type annotation.